MANIFEST.in

```diff
@@ -1,5 +1,7 @@
 include .codecov.yml
 include .coveragerc
+include .flake8
+include .pre-commit-config.yaml
```

Comment:
```suggestion
include .pre-commit-config.yaml
```

Reply:
pre-commit doesn't work without the .git directory so there's no use having it in the sdist